src/components/TransactionList.vue

```diff
@@ -99,6 +99,7 @@ import { isProxyData, ProxyType, ProxyTransactionDirection } from '../lib/ProxyD
 import { createCashlink } from '../hub';
 import { useConfig } from '../composables/useConfig';
 import { useWindowSize } from '../composables/useWindowSize';
+import { useTransactionInfo } from '../composables/useTransactionInfo';
```

Comment:
Can these composables also be used in the Transaction modals? `TransactionModal`, `BtcTransactionModal` and `UsdcTransactionModal`. I believe there the same computations are made?

Reply:
Good catch, I have replaced all the duplicated code. Not as much as in the TransactionList but still a good amount 👍🏼